Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN apk add --no-cache -t build-dependencies \
     uwsgi \
     uwsgi-python3 \
     brotli \
- && pip3 install --no-cache -r requirements.txt \
+ && pip3 install --break-system-packages --no-cache -r requirements.txt \
```

Comment:
Because the new pip refuses to install the packages without a virtualenv.
Since this is docker image, there is no issue to break the system packages ( = install Python packages not from Alpine).